tests/src/test/java/org/eclipse/hono/tests/amqp/AmqpUploadTestBase.java

```diff
@@ -13,6 +13,7 @@
 package org.eclipse.hono.tests.amqp;
 
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.eclipse.hono.tests.IntegrationTestSupport.PARAMETERIZED_TEST_NAME_PATTERN;
```

Comment:
Can we please not statically import constants? This makes it very hard to know where it comes from when reading code that makes use of them ...

Reply:
I've changed that. I agree with Dejan (other PR comment) that we should create a Checkstyle rule for that then.